Not sure what's going on here, but to fix enlightenment build I've
switched the data types to the ones defined in khrplatform.h.

--- src/lib/evas/Evas_GL.h.orig	2014-11-11 11:26:36.753709613 +0100
+++ src/lib/evas/Evas_GL.h	2014-11-11 11:27:01.946647835 +0100
@@ -843,8 +843,8 @@ EAPI Evas_GL_Surface         *evas_gl_cu
  *-----------------------------------------------------------------------*/
 
 #ifndef KHRONOS_SUPPORT_INT64
-typedef unsigned long long khronos_uint64_t;
-typedef signed long long   khronos_int64_t;
+typedef uint64_t khronos_uint64_t;
+typedef int64_t khronos_int64_t;
 #endif
 
 // Due to build conflicts on various platforms, we can't use GL[u]int64 directly
